### ISSUE TITLE
Hydra twitter brute force

### ISSUE THREAD

**Issue by MehmetAliCan10** [vanhauser-thc/thc-hydra#906]
Hi

Twitter How can I bruteforce with Hydra and starting from 8, all letters, numbers, symbols, uppercase letters...?

I tried it, it doesn't work, please help?

hydra -l [username@hotmail.com](mailto:username@hotmail.com) -x:8:8:[abcdefghijklmnopqrsşŞtuvwxyzåäöABCDEFGHIJKLMNOPQRSTUVWXYZÅÄÖ0123456789!@#$%^&*()-_+=~`[]{}|\:;"'<>,.?/ ] https://www.twitter.com/ https-post-form "/:username=^USER^&password=^PASS^:F=incorrect" -V

I do it, it doesn't work... ?

**Comment by MehmetAliCan10** [comment]
Yes but i can do shorter but how to twitter command Hydra login ?

**Comment by MASTREX** [comment]
On sites like twitter google or Facebook you cannot just brute force to crack an account

**Comment by MehmetAliCan10** [comment]
But someone did brute forced hacked password found! ...

**Comment by MehmetAliCan10** [comment]
But my command not work...

**Comment by MASTREX** [comment]
> But my command not work...

Yeah its wrong, you have use some kind of proxy tool to look how the data is sent to the server,l. There definitely will be some headers and cookies. Also double check how the failed string look like

**Comment by MehmetAliCan10** [comment]
> > But my command not work...
> 
> Yeah its wrong, you have use some kind of proxy tool to look how the data is sent to the server,l. There definitely will be some headers and cookies. Also double check how the failed string look like

With burpsuite?

**Comment by MASTREX** [comment]
Yes you can use burpsuit

**Comment by MehmetAliCan10** [comment]
but hydra is faster than burpsuit brute force? but Hydra more false positives?

**Comment by MehmetAliCan10** [comment]
> Yes you can use burpsuit

but hydra is faster than burpsuit brute force? but Hydra more false positives?

**Comment by MASTREX** [comment]
> > Yes you can use burpsuit
> 
> but hydra is faster than burpsuit brute force? but Hydra more false positives?
> 
> 
> 

Burpsuit has some predefined time and rate limits but hydra doesn't

**Comment by MehmetAliCan10** [comment]
> Yes you can use burpsuit

Have you email please me contact?

**Comment by MehmetAliCan10** [comment]
hydra -l [username@hotmail.com](mailto:username@hotmail.com) -x:8:8:[abcdefghijklmnopqrsşŞtuvwxyzåäöABCDEFGHIJKLMNOPQRSTUVWXYZÅÄÖ0123456789!@#$%^&*()-_+=~`[]{}|:;"'<>,.?/ ] https://www.twitter.com/ https-post-form "/:username=^USER^&password=^PASS^:F=incorrect" -V


It's not work someone me help please?

**Comment by R-Rothrock** [comment]
Maybe take a look at this: https://github.com/vanhauser-thc/thc-hydra/issues/612

Also, you should probably keep in mind that Twitter might not allow you to put all of those characters in a password/username.
